tests/beaker_tests/cisco_bgp_neighbor_af/test_bgpneighboraf.rb

```diff
@@ -539,154 +604,170 @@ def build_manifest_bgp_nbr_af(tests, id)
 
 # Wrapper for bgp_nbr_af specific settings prior to calling the
 # common test_harness.
-def test_harness_bgp_nbr_af(tests, id)
+def test_harness_bgp_nbr_af(tests, id, platform)
   af = af_title_pattern_munge(tests, id, 'bgp_neighbor_af')
   logger.info("\n--------\nTest Case Address-Family ID: #{af}")
 
-  # Set up remote-as if necessary
-  bgp_nbr_remote_as(agent, tests[id][:remote_as]) if tests[id][:remote_as]
-
   tests[id][:ensure] = :present if tests[id][:ensure].nil?
-  tests[id][:show_pattern] = af_pattern(tests, id, af)
   tests[id][:resource_cmd] = puppet_resource_cmd(af)
 
-  # Build the manifest for this test
-  build_manifest_bgp_nbr_af(tests, id)
+  if platform == 'ios_xr' and af[:vrf] != 'default'
+    logger.info("\n--------\nSkip Case Address-Family ID: #{af} for ios_xr")
```

Comment:
On XR, I tried to configure manually. Two configuration works:

1)
router bgp 1
 address-family ipv4 unicast
 neighbor 1.1.1.1
 remote-as 2                  <<<< this is required
 address-family ipv4 unicast  <<<< parent-child address-family must match

2).
router bgp 1
 bop router-id 1.2.3.4
 address-family ipv4 unicast
 vrf blue
  rd auto
  neighbor 1.1.1.1
   remote-as 2               <<<< this is optional
                             <<<< can not configure address-family

For non-default VRF, address-family is not allowed.